djangoProject3/my-bookstore/src/components/Users.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+
+function UserList() {
+    const [users, setUsers] = useState([]);
+    const [orderData, setOrderData] = useState(null);
+    const [orderIdInput, setOrderIdInput] = useState('');
+    const [orderNotFound, setOrderNotFound] = useState(false);
+    const token = sessionStorage.getItem('authToken');
+
+    useEffect(() => {
+        if (!token) {
+            return;
+        }
+        fetch('http://localhost:8000/user/', {
```

Comment:
this works but can you try using Axios as it is used for making requests as industry standard

Reply:
i try it but it doesn't work:(. I will try again